spec/system/admin/homepage/homepage_spec.rb

```diff
@@ -22,7 +22,7 @@
     scenario "Admin menu links to homepage path" do
       visit new_admin_widget_card_path(header_card: true)
 
-      click_link Setting["org_name"] + " Administration"
+      click_link("CONSUL logo")
```

Comment:
Strange that this test passes :thinking:. I guess we'll have to change the `layouts.header.logo` i18n key so it says the same as what's present in the image :thinking:.